app/src/main/java/org/xwiki/android/sync/activities/UserListAdapter.kt

```diff
@@ -87,6 +90,7 @@ class UserListAdapter(private var searchResults: List<ObjectSummary>)
         diffResult.dispatchUpdatesTo(this)
         searchResults = listOf()
         this.searchResults = results
+        (context as SyncSettingsActivity).scrollToCurrentPosition()
```

Comment:
It is not the best way. Even if you want to use casts (which is not best practice) - use `as?` to be sure that here will not be thrown some exception like ClassCastException

Reply:
I can put "?" safe call but the context here is non-null. So is it necessary?